README.md

```diff
@@ -30,20 +65,246 @@ public function registerBundles()
+
+In this exemple, I have add a new folder for store all articles.
+
+For get the location id of the parent eZ Platform content, go to the admin UI and select the future parent content, click on the details tabs, and read the "Location id" like this:
+
+![parent folder](src/Resources/doc/dest_folder.jpg)
+
+> Note: the best practice is to define this parent id in your `parameter.yml` file or your `.env.local` file for each execution environment.
+
+```php
+// In your DataflowType
+
+use CodeRhapsodie\EzDataflowBundle\Writer\ContentWriter;
+use CodeRhapsodie\DataflowBundle\DataflowType\AbstractDataflowType;
+use eZ\Publish\API\Repository\ContentService;
+[...]
+
+class MyDataflowType extends AbstractDataflowType
+{
+    //[...]
+    /**
+     * @var ContentWriter
+     */
+    private $contentWriter;
+
+    /**
+     * @var ContentService
+     */
+    private $contentService;
+
+    public function __construct(ContentWriter $contentWriter, ContentService $contentService)
+    {
+        $this->contentWriter = $contentWriter;
+        $this->contentService = $contentService;
+    }
+    //[...]
+    protected function buildDataflow(DataflowBuilder $builder, array $options): void
+    {
+        //[...]
+        $builder->addStep(function ($data) {
+            if (!isset($data['id'])) {
+                return false;
+            }
+
+            $remoteId = sprintf('article-%d', $data['id']);
+            unset($data['id']);
+
+            $parent_location_for_new_article = 54;
+
+            $lang = 'eng-GB';
+            try {
+                $content = $this->contentService->loadContentByRemoteId($remoteId);
+                $struct = ContentUpdateStructure::createForContentId($content->id, $lang, $data);
+            } catch (NotFoundException $e) {
+                $struct = new ContentCreateStructure(
+                    'article',
+                    $lang
+                    [$parent_location_for_new_article],
+                    $data,
+                    $remoteId,
+                );
+            }
+
+
+            return $struct;
+        });
+        $builder->addWriter($this->contentWriter);
+    }
+}
+```
+
+# Admin UI
+
+## Access to the eZ Dataflow UI
+
+You can access to the eZ Dataflow administation UI from your eZ Platform admin back-office.
+
+![Admin menu](src/Resources/doc/ez_dataflow_admin_menu.jpg)
+
+1. Click to "Admin"
+1. Click to "eZ Dataflow"
+
+## Add a new schedule
+
+Go to the eZ Dataflow admin, and click on the "+" orange button. 
+
+Il the new popin, complete the fields:
+
+![Add a new schedule](src/Resources/doc/add_new_schedule.jpg)
+
+1. Type the Dataflow name
+1. Select the Dataflow type. The list is automaticaly generated from the list of Symfony service with the tags `coderhapsodie.dataflow.type`. If your dataflow type is not present, [check the configuration](https://github.com/code-rhapsodie/dataflow-bundle#check-if-your-dataflowtype-is-ready)
```

Comment:
> The list is automaticaly generated from the list of Symfony service with the tags

The list is automatical**l**y populated with Symfony services tagged with `coderhapsodie.dataflow.type`